app/mailers/proposal_mailer.rb

```diff
@@ -35,9 +35,11 @@ def proposal_complete(proposal)
     )
   end
 
-  def proposal_updated_no_action_required(user, proposal, modifier = nil)
+  def proposal_updated_no_action_required(user = User.last, proposal = Proposal.last, modifier = nil)
```

Comment:
Shoot- that was for testing :) I'll fix!

--
Leonard Bogdonoff
949-322-8496 | @rememberlenny

> On Mar 9, 2016, at 6:40 PM, Jessie A. Young <notifications@github.com> wrote:
> 
> In app/mailers/proposal_mailer.rb:
> 
> > @@ -35,9 +35,11 @@ def proposal_complete(proposal)
> >      )
> >    end
> >  
> > -  def proposal_updated_no_action_required(user, proposal, modifier = nil)
> > +  def proposal_updated_no_action_required(user = User.last, proposal = Proposal.last, modifier = nil)
> we don't want to set default values for this. Since it is the actual email, we want to make sure we are always passing in the user and proposal . we have a default value of nil for modifier because sometimes will update a proposal via rails console in which case there will be no recorded modifier.
> 
> —
> Reply to this email directly or view it on GitHub.
>